kobo/settings/base.py

```diff
@@ -1707,23 +1708,11 @@ def dj_stripe_request_callback_method():
     os.environ.get('SUPPORT_BRIEFCASE_SUBMISSION_DATE') != 'True'
 )
 
-DEFAULT_VALIDATION_STATUSES = [
-    {
-        'uid': 'validation_status_not_approved',
-        'color': '#ff0000',
-        'label': 'Not Approved'
-    },
-    {
-        'uid': 'validation_status_approved',
-        'color': '#00ff00',
-        'label': 'Approved'
-    },
-    {
-        'uid': 'validation_status_on_hold',
-        'color': '#0000ff',
-        'label': 'On Hold'
-    },
-]
+DEFAULT_VALIDATION_STATUSES = {
+    'validation_status_not_approved': 'Not Approved',
+    'validation_status_approved': 'Approved',
+    'validation_status_on_hold': 'On Hold',
+}
```

Comment:
Indeed, the front end never used the color provided by the back end. 
When we wrote this part of code (the validation status feature), the idea behind this was to have a specific setting which could override the setting. Each time I looked at it, I found it pretty complicated to understand. If we were to implement that on the front end, we would to overwrite on the fly the colors for existing data (because colors are hard-coded in Mongo).
Long story short, I'll fix the API docstring and formpack test fixtures.